taxcalc/utils.py

```diff
@@ -2,6 +2,8 @@
 import pandas as pd
 from pandas import DataFrame
 from collections import defaultdict
+import pytest
```

Comment:
@talumbau said:
> I didn't have a chance to comment before this got merged. What is the reason that pytest is imported here? I don't think it should be imported into library code, only test code.

Thanks for pointing out this oversight.  I'll remove that unused import statement.